frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -55,13 +55,17 @@ export const getCephSC = (scData: K8sResourceKind[]): K8sResourceKind[] =>
     );
   });
 
+export const getOperatorVersion = (operator: K8sResourceKind): string => {
+  return operator?.status?.installedCSV;
+};
+
```

Comment:
```suggestion
export const getOperatorVersion = (operator: K8sResourceKind): string => operator?.status?.installedCSV;

```

Reply:
yeah... okay... I missed this in your previous suggestion...